packages/plugin-defender/package.json

```diff
@@ -0,0 +1,40 @@
+{
+  "name": "@openzeppelin/defender-upgrades",
+  "version": "1.4.3",
+  "description": "",
+  "repository": "https://github.com/OpenZeppelin/openzeppelin-upgrades/tree/master/packages/plugin-defender",
+  "license": "MIT",
+  "main": "dist/index.js",
+  "files": [
+    "/dist",
+    "/src"
+  ],
+  "scripts": {
+    "prepublish": "rimraf dist *.tsbuildinfo",
+    "prepare": "tsc -b",
+    "test": "tsc -b && bash scripts/test.sh",
+    "test:watch": "fgbg 'bash scripts/test.sh --watch' 'tsc -b --watch' --",
+    "version": "node ../../scripts/bump-changelog.js"
+  },
+  "devDependencies": {
+    "@nomiclabs/hardhat-ethers": "^2.0.0",
+    "@openzeppelin/hardhat-upgrades": "^1.4.2",
+    "@openzeppelin/upgrades-core": "^1.4.2",
+    "@types/mocha": "^7.0.2",
+    "ava": "^3.9.0",
+    "ethers": "^5.0.5",
+    "fgbg": "^0.1.4",
+    "hardhat": "^2.0.2",
+    "promisified": "^0.5.0",
+    "rimraf": "^3.0.2"
+  },
+  "dependencies": {
+    "defender-admin-client": "^1.3.0"
+  },
+  "peerDependencies": {
+    "@nomiclabs/hardhat-ethers": "^2.0.0",
+    "@openzeppelin/hardhat-upgrades": "^1.4.2",
+    "@openzeppelin/upgrades-core": "^1.4.2",
+    "hardhat": "^2.0.2"
+  }
```

Comment:
As we discussed, we want to make `@openzeppelin/hardhat-upgrades` a full dependency of this package, and eventually have a different package for Truffle-Defender integration.

The other dependencies here don't need to be listed.